news/2016-10-10-tip2-devmode.asciidoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: news
+title: Weld Tip &num;2 - Development Mode
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Meet the Weld Probe - a development tool which allows to inspect CDI components at runtime.
```

Comment:
I'd say this goes without `the` as long as you are reffering to something by its name -> `Meet Weld Probe`